lib/modem_info/modem_info.c

```diff
@@ -110,13 +127,34 @@ static const struct modem_info_data operator_data = {
 	.data_type = AT_PARAM_TYPE_STRING,
 };
 
+static const struct modem_info_data mcc_data = {
+	.cmd = AT_CMD_CURRENT_OP,
+	.param_index = OPERATOR_PARAM_INDEX,
+	.param_count = OPERATOR_PARAM_COUNT,
```

Comment:
Is there a way to split MCC and MNC? Curently it looks like both will hold the entire MNC_MCC string.

Edit: I see there's a function doing this further down...

Reply:
This struct definition is understandably confusing. We should change this architecture in the future, as we shouldn't need to define two similar structs if we want multiple data from the same AT-command.